File: Ejercicios Laragon/proyectopython2644590/age.py
edad = int(input("Input your age: "))

if edad < 0:
    print("Plase, input a valid data! ")
elif edad <= 12:
    print("You're a child")
elif edad <= 21:
    print("You're a teenager")
elif edad <= 31:
    print("You're a young person")
elif edad <= 60:
    print("You're an adult")
else:
    print("You're an older adult")

File: Ejercicios Laragon/proyectonodejs2644590/Age Project/age.js
function verPoblacion(){

    let edad = document.getElementById("edad").value
    let mensaje = document.getElementById("mensaje")

    if (edad <= 0){
        mensaje.innerHTML = "Please, input a valid data."
    }else if(edad <= 12){
        mensaje.innerHTML = "You're a child";
    }else if(edad <= 21){
        mensaje.innerHTML =  "You're a teenager";
    }else if(edad <= 31){
        mensaje.innerHTML =  "You're a young person";
    }else if(edad <= 60){
        mensaje.innerHTML =  "You're an adult";
    }else{
        mensaje.innerHTML =  "You're an older adult";
    }  
    
} 
     